core/integration/git_test.go

```diff
@@ -37,3 +37,33 @@ func TestGit(t *testing.T) {
 	require.NoError(t, err)
 	require.Contains(t, res.Git.Branch.Tree.File.Contents, "Dagger")
 }
+
+func TestGitKeepGitDir(t *testing.T) {
+	t.Parallel()
+
+	res := struct {
+		Git struct {
+			Branch struct {
+				Tree struct {
+					Entries []string
+					File    struct {
+						Contents string
+					}
+				}
+			}
+		}
+	}{}
+
+	err := testutil.Query(
```

Comment:
@jaredpatrick37 You can use the generated Go SDK to do this testing! 

See an example from @vito here:

https://github.com/dagger/dagger/blob/f6893d995b68849c6fd2a95abf0490ab7bcfa870/core/integration/file_test.go#L117-L121

Reply:
Updated